site/public/js/server.js

```diff
@@ -150,14 +150,14 @@ function changeDiffView(div_name, gradeable_id, who_id, version, index, autochec
  * @param {object} loadingTools - loading tools used by span containing the text that 
  *                                correspond to the state of the test case (show, hide, loading)
  */
-function CollapseTestcaseOutput(div_name, index, loadingTools){
-    $("#show_char_" + index).toggle();
-    $("#" + div_name).empty();
-    toggleDiv(div_name);
+// function CollapseTestcaseOutput(div_name, index, loadingTools){
```

Comment:
Is there a reason why these are commented out?

Reply:
I made this commit to show someone my code, so the commented out section was a byproduct of my work flow.  It was  removed in my latest commit.